js/components/GoOnline.tsx

```diff
@@ -109,8 +109,8 @@ export class GoOnline extends React.Component<IGoOnlineProps, IGoOnlineState> {
         const latestAuthenticationToken = manywho.state.getAuthenticationToken(this.props.flowKey);
         if (this.flow) {
             requests = this.flow.requests.map((request, index) => {
-                request.stateId = this.flow.state.id;
-                request.stateToken = this.flow.state.token;
+                request.request.stateId = this.flow.state.id;
```

Comment:
This has introduced quite a few references to request.request which is a bit confusing. Is it possible to rename something to prevent this?

Reply:
Done